extensions/csrc/kernel/cuda/flash_decoding_attention_kernel.cu

```diff
@@ -34,11 +33,25 @@ constexpr unsigned int nextHighestPowerOf2(unsigned int v) {
   return v;
 }
 
+template <typename T>
```

Comment:
Why do you want to move 'zero' operation here?

Reply:
Just be used here